app/src/main/java/com/example/pygmyhippo/user/ProfileFragment.java

```diff
@@ -88,6 +133,140 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         // need to do this so the listener is connected
         role_dropdown.setOnItemSelectedListener(this);
 
+
+        ImageView editButton = root.findViewById(R.id.E_P_edit);
+        Button updateButton = root.findViewById(R.id.E_P_update);
+
+        // All the text fields
+        EditText name_f = root.findViewById(R.id.E_P_textName);
```

Comment:
what does the f mean?

Reply:
I'm also curious why you choose to use snake case for these variables

Reply:
f for field